tests/__init__.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from tests.fixtures.environment_variables import *
+from message_passing_nn.utils.postgres_connector import PostgresConnector
+
+os.environ['DATASET_TABLE'] = TEST_DATASET
+os.environ['PENALTY_TABLE'] = TEST_PENALTY
+os.environ["DATABASE"] = PDB_TEST
+os.environ["POSTGRES_USERNAME"] = POSTGRES
+os.environ["POSTGRES_PASSWORD"] = POSTGRES
+os.environ["POSTGRES_HOST"] = LOCALHOST
+os.environ["POSTGRES_PORT"] = PORT
+
```

Comment:
Should this not be in a config file

Reply:
No, it's in the tests. Not really a need for that